README.md

```diff
@@ -100,6 +101,10 @@ what](https://chris.beams.io/posts/git-commit/).
 Follow linter config (open a PR with rule changes if you disagree) and write
 tests according to [Better Specs](http://www.betterspecs.org).
 
+For react files this project uses Prettier for stylistic rules and ESlint for code quality rules. To avoid conflicts, ESlint runs Prettier as an ESlint rule. 
```

Comment:
```suggestion
For React files this project uses Prettier for stylistic rules and ESlint for code quality rules. To avoid conflicts, ESlint runs Prettier as an ESlint rule. 
```